From 7e389b67c24dfe4466112c988b240c807e7e2414 Mon Sep 17 00:00:00 2001
From: "petrak@" <petrathekat@gmail.com>
Date: Thu, 4 Jan 2024 15:49:50 -0500
Subject: [PATCH 165/188] Add auto-pairs to scheme language support (#9232)

Currently, typing a single quote in a `.scm` file "helpfully" auto-
completes a closing quote. This is because there is no auto-pairs
section in the languages.toml. This commit adds that.
---
 languages.toml | 6 ++++++
 1 file changed, 6 insertions(+)

--- a/languages.toml
+++ b/languages.toml
@@ -1914,6 +1914,12 @@ shebangs = ["scheme", "guile", "chicken"
 comment-token = ";"
 indent = { tab-width = 2, unit = "  " }
 
+[language.auto-pairs]
+'(' = ')'
+'{' = '}'
+'[' = ']'
+'"' = '"'
+
 [[grammar]]
 name = "scheme"
 source = { git = "https://github.com/6cdh/tree-sitter-scheme", rev = "af3af6c9356b936f8a515a1e449c32e804c2b1a8" }
